### ISSUE TITLE
Tests are failing due to OpenGL installation in `test_and_deploy.yaml`

### ISSUE THREAD

**Issue by jo-mueller** [BiAPoL/napari-crop#28]
I'm not sure whether this installation is necessary for packages that do not rely on GPU-functionality. I'll start a PR to see if tests still pass after removing this part from `test_and_deploy.yaml`.

**Comment by haesleinhuepf** [comment]
Napari needs OpenGL

Consider replacing "git" in this line with "https":
https://github.com/BiAPoL/napari-crop/blob/main/.github/workflows/test_and_deploy.yml#L49

See also:
https://github.com/napari/cookiecutter-napari-plugin/pull/104

**Comment by jo-mueller** [comment]
Ugh, it's that one. Thanks for pointing it out!

**Comment by jo-mueller** [comment]
Hm...I changed the actions so that they don't rely on the OpenGL installation anymore, but rather install conda. Also, tests now run with pytest directly rather than tox.

I think the error in the github actions is the same one I recently saw in [napari stress](https://github.com/BiAPoL/napari-stress/issues/93). There, removing Python 3.7 from the list of supported Python versions solved the problem.

Of course not really the problem, but only the symptoms.
